src/module_compilation.ml

```diff
@@ -46,7 +46,10 @@ let build_cm cctx ~dep_graphs ~precompiled_cmi ~cm_kind (m : Module.t) =
         (* symlink the .cmi into the public interface directory *)
         if Module.visibility m <> Visibility.Private
         && (Obj_dir.need_dedicated_public_dir obj_dir) then
-          SC.add_rule sctx ~sandbox:false ~dir
+          SC.add_rule sctx
+            (* CR-someday aalekseyev: why do we have [no_sandboxing] here? *)
```

Comment:
We are just doing a syscall without calling out to an external command. We can reasonably expect that sandboxing is not necessary. Though, it seems fine to remove the argument altogether

Reply:
I added a function `Action.is_useful_to_sandbox` to help with that.